data/json/items/gun/762R.json

```
@@ -114,5 +114,43 @@
       [ "underbarrel", 1 ]
     ],
     "relative": { "price": 10000 }
+  },
+  {
+    "id": "psl",
+    "copy-from": "rifle_semi",
+    "looks_like": "ar15",
+    "//": "Mom can we have a McDragunov? Mom: we have McDragunov at home. The McDragunov at home:",
+    "type": "GUN",
+    "name": { "str": "PSL" },
+    "description": "The Romanian PSL is a designated marksman's rifle developed by Romania after political tensions with Soviet Russia forced them to develop their own alternative to the SVD.",
+    "weight": "4100 g",
+    "volume": "2167 ml",
+    "longest_side": "87 cm",
+    "price": 290000,
+    "price_postapoc": 4000,
+    "to_hit": -1,
+    "bashing": 12,
+    "material": [ "steel", "wood" ],
+    "color": "brown",
+    "ammo": [ "762R" ],
+    "dispersion": 150,
+    "durability": 9,
+    "modes": [ [ "DEFAULT", "semi-auto", 1 ] ],
+    "valid_mod_locations": [
+      [ "accessories", 4 ],
+      [ "barrel", 1 ],
+      [ "bore", 1 ],
+      [ "brass catcher", 1 ],
+      [ "grip", 1 ],
+      [ "mechanism", 4 ],
+      [ "muzzle", 1 ],
+      [ "rail mount", 1 ],
+      [ "sights mount", 1 ],
```

```suggestion
      [ "sights mount", 1 ],
```
I don't think a DMR is going to need you to add a mount to it for a scope(esp. as Century includes an optic with their imports of it https://www.centuryarms.com/psl-54c-semi-auto-rifle-cal-7-62x54r.html) 🤔

---

Fair enough. I was thinking of just a bare, unmodified PSL, but eh, works for me. ¯\_(ツ)_/¯